### PR TITLE
* BerayBentesen.cs: Author created

### PR DESCRIPTION
**Pull request by beraybentesen**
* Web.config:
* Firehose.Web.csproj:

If you are submitting a new blog please read:

By sending this pull request to add my blog I verify that I adhear to the Planet Xamarin blog guidelines:

- I have a valid blog & RSS URL
- Host NO malicious or offensive content on the blog (including photos, swearing, etc.)
- Blog is active with atleast 3 blog posts in the last 6 months
- If the blog has mixed content (Xamarin and Personal/Non-Xamarin blogs) a filter has been applied
- If you delete your blog you will come delete your blog from Planet Xamarin
- Your blog may be removed at any time if any of these are broken.

### CODE REVIEW

**Comment by beraybentesen** [comment]
Reverted changes. I think Xamarin Studio removed those lines itself.

File: src/Firehose.Web/Firehose.Web.csproj [hunk 1]
```diff
@@ -307,8 +307,6 @@
     <Compile Include="Authors\StefanDeVogelaere.cs" />
     <Compile Include="Authors\AlexandrSorokoletov.cs" />
     <Compile Include="ViewModels\PreviewViewModel.cs" />
-    <Compile Include="Authors\IoneSouzaJunior.cs" />
```
Comment (Cheesebaron) [inline src/Firehose.Web/Firehose.Web.csproj]: Why did you remove these two authors?

File: src/Firehose.Web/Web.config [hunk 2]
```diff
@@ -37,10 +36,10 @@
   <system.web>
     <compilation debug="true" targetFramework="4.5.1">
       <assemblies>
-        <add assembly="System.Globalization, Version=4.0.0.0,
-                       Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" />
-        <add assembly="System.Runtime, Version=4.0.0.0,
-                       Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" />
+		<add assembly="System.Globalization, Version=4.0.0.0,
```
Comment (Cheesebaron) [inline src/Firehose.Web/Web.config]: Indentation is wrong

File: src/Firehose.Web/Firehose.Web.csproj [hunk 3]
```diff
@@ -307,8 +307,6 @@
     <Compile Include="Authors\StefanDeVogelaere.cs" />
     <Compile Include="Authors\AlexandrSorokoletov.cs" />
     <Compile Include="ViewModels\PreviewViewModel.cs" />
```
Comment (Cheesebaron) [inline src/Firehose.Web/Firehose.Web.csproj]: You forgot to add yourself here...

**Comment by Cheesebaron** [comment]
Closing this as there is no response from @beraybentesen
